test/emailpassword/signupFeature.test.js

```diff
@@ -412,7 +412,12 @@ describe(`signupFeature: ${printPath("[test/emailpassword/signupFeature.test.js]
                 appName: "SuperTokens",
                 websiteDomain: "supertokens.io",
             },
-            recipeList: [EmailPassword.init(), Session.init()],
+            recipeList: [
+                EmailPassword.init(),
+                Session.init({
+                    enableAntiCsrf: true,
```

Comment:
Is there a better way to set this to true for tests as opposed to changing every single test?